Source/Scene/Cesium3DTile.js

```diff
@@ -994,14 +990,26 @@ define([
 
     function applyDebugSettings(tile, tileset, frameState) {
         var hasContentBoundingVolume = defined(tile._header.content) && defined(tile._header.content.boundingVolume);
+        var empty = tile.hasEmptyContent || tile.hasTilesetContent;
 
         var showVolume = tileset.debugShowBoundingVolume || (tileset.debugShowContentBoundingVolume && !hasContentBoundingVolume);
         if (showVolume) {
+            var color;
+            if (!tile._finalResolution) {
+                color = Color.YELLOW;
+            } else if (hasContentBoundingVolume) {
+                color = Color.WHITE;
+            } else if (empty) {
+                color = Color.GREEN;
+            } else {
+                color = Color.RED;
+            }
```

Comment:
A little picky, but since red usually means "bad" and green usually mean "good" or "go", I would mark the colors something like so:

`!finalResolution` -> Yellow
`empty` -> gray
`hasContentBoundingVolume` -> green
otherwise -> blue

Reply:
I ended up going with:

`!finalResolution` -> yellow
`empty` -> gray
`boundingVolume` -> white
`contentBoundingVolume` -> blue (the code for drawing content bounding volumes is further down)

I like `white` as the "base" color because it's really neutral.